Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,72 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render a `<picture>` element for responsive image support.
+public struct Picture: Component {
+
+    /// Image to use as fallback.
+    public var image: Image
+    /// A set of `source` elments from which a user agent will chose based on its requirements.
+    public var sources: [Source]
+
+    /// Initialize a `Picture` component with multiple `source` elements.
+    internal init(image: Image, sources: [Picture.Source] = []) {
```

Comment:
This initializer can also be removed, right? Given that the default, memberwise initializer can be used within the public one. But I'm also thinking if this shouldn't be the default public initializer. Isn't it more common to use multiple sources when using a `<picture>` element? Otherwise, you might as well just use a plain `<img>`, or perhaps I'm missing something?

Reply:
It is more common, but the spec does allow a single source. Examples would be using a `srcset` attribute with multiple images and a `sizes` attribute with the required sizes, or using an advanced image `type` (like AVIF), and a JPG fallback image.

Reply:
So, here's what I think we should do:

1. Make this initializer (the one that takes an array of `sources`) `public`, since it covers the majority use case.
2. Remove the initializer that takes a single `source` (since that same functionality can easily be achieved by simply passing a single `source` within an array).
3. Add parameter documentation to the initializer (similar to what the other built-in components have).